front/src/draw/freqAxis.ts

```diff
@@ -71,8 +78,18 @@ export default function drawTimeAxis(
     });
   });
 
+  mayorTicks.forEach((freq) => {
+    const y = (height * (max - freq)) / range;
+    drawFreqTick(ctx, y, {...style, ...MAYOR_TICK_STYLE_GRID})
+  })
+
```

Comment:
Would be great to add an argument allowing the user to choose to view gridlines or not,
and default to not showing them (as they are not shown in the original plot).